### ISSUE TITLE
Problem while executing code 

### ISSUE THREAD

**Issue by mishpri0209** [xdeng7/scale-aware_da#3]
Hi @xdeng7 
I'm facing problem in **scalenet_Deeplab.py** file : **Sizes of tensors must match except in dimension 2. Got 16 and 32 (The offending index is 0)**

 def forward(self, input):
        x, low_level_feat,l1,l2,l3,l4= self.backbone(input)
        # import pdb
        # pdb.set_trace()

        x1=self.avg_pool(l1)
        print('x1:', x1.shape)
        x2=self.avg_pool(l2)
        print('x2:', x2.shape)
        x3=self.avg_pool(l3)
        print('x3:', x3.shape)
        x4=self.avg_pool(l4)
        print('x4:', x4.shape)        
        x = self.aspp(x)
        print('x:', x.shape)

        **x=torch.cat((x4,x3,x2,x1,x),dim=1)**
       cout1=x
        x,_=self.ca(x)
        x = self.decoder(x, low_level_feat)
        # feature=x
        x = F.interpolate(x, size=input.size()[2:], mode='bilinear', align_corners=True)

        return x, cout1

Can you please let me know how to deal with this problem.
Thanks in advance

Priyanka

**Comment by mishpri0209** [comment]
Hi , Kindly reply , I'm kind of stuck here .
I have exactly taken your code and trying to execute .


Thanks 
Priyanka
